packages/create-gasket-app/lib/config/utils.js

```diff
@@ -0,0 +1,32 @@
+import { default as diagnostics } from 'diagnostics';
+const debug = diagnostics('gasket:cli:config:utils');
+
+/**
+ * Returns specified env option if set or appropriate fallback
+ *
+ * @param {Object} options - CLI Option
+ * @param {string} commandId - Name of the command
+ * @param {function} warn - Warning logger
+ * @returns {string} environment
+ */
+export function getEnvironment(options, commandId, warn) {
```

Comment:
Env should not be important for `create-gasket-app` I wouldn't think. I do see where this function is used. Can this file just be deleted?